packages/react-core/src/components/FormCore/__tests__/__snapshots__/useControlledField.spec.ts.snap

```diff
@@ -4,7 +4,7 @@ exports[`useControlledField returns the expected values in the happy path 1`] =
 {
   "errorMessage": undefined,
   "hasError": false,
-  "invalid": true,
+  "invalid": false,
```

Comment:
Updated due to difference in behavior on removal of `waitForNextUpdate`, does not impact production code